src/_bentoml_impl/server/app.py

```diff
@@ -36,6 +37,31 @@
 R = t.TypeVar("R")
 
 
+LIMITER_MAP: dict[str, t.Any] = {}
```

Comment:
It's suboptimal to use a global map, instead I would prefer to store this map inside the app factory.

Reply:
makes more sense since we need service level